install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -1,5 +1,8 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+  ack-4.12-kube-1.25-api-removals-in-4.13: |-
```

Comment:
hmm.. in previous versions, the gate was 
```
ack-4.11-kube-1.25-api-removals-in-4.12: 
Kubernetes 1.25 and therefore OpenShift 4.12
```
```
ack-4.8-kube-1.22-api-removals-in-4.9:
Kubernetes 1.22 and therefore OpenShift 4.9
```
while in this case we have
```
ack-4.12-kube-1.25-api-removals-in-4.13
Kubernetes 1.26 and therefore OpenShift 4.13 
```
while the latter imo is more semantically correct, considering that afaik we're removing some 1.25 api, not 1.26, however it is inconsistent with previous versions. however I personally don't think it will cause much confusion, so mentioning it just in case.
cc @shellyyang1989 wdyt?

Reply:
sidenote: the mentioned knowledge article [1] mentions:
 oc -n openshift-config patch cm admin-acks --patch '{"data":{"**ack-TBD-CHANGETHIS**":"true"}}' --type=merge
 is expected to be updated later, after this PR merges right?

[1]https://access.redhat.com/articles/6958394

Reply:
> hmm.. in previous versions, the gate was
> 
> ```
> ack-4.11-kube-1.25-api-removals-in-4.12: 
> Kubernetes 1.25 and therefore OpenShift 4.12
> ```
> 
> ```
> ack-4.8-kube-1.22-api-removals-in-4.9:
> Kubernetes 1.22 and therefore OpenShift 4.9
> ```
> 
> while in this case we have
> 
> ```
> ack-4.12-kube-1.25-api-removals-in-4.13
> Kubernetes 1.26 and therefore OpenShift 4.13 
> ```
> 
> while the latter imo is more semantically correct, considering that afaik we're removing some 1.25 api, not 1.26, however it is inconsistent with previous versions. however I personally don't think it will cause much confusion, so mentioning it just in case. cc @shellyyang1989 wdyt?

Right, imho we need to keep them consistent. So 1.26 sounds right to me!

Reply:
Thank you for pointing out the typo. Fixed it.